core/tools/blueos_startup_update/blueos_startup_update.py

```diff
@@ -186,31 +186,25 @@ def boot_config_filter_conflicting_configuration_at_session(
 
 def load_file(file_name) -> str:
     command = f'cat "{file_name}"'
-    output = run_command(command, False)
-    logger.info(output)
-    return output.stdout
+    return run_command(command, False, log_output=False).stdout
```

Comment:
I'm not a big fan of removing the log for such functions

Reply:
sounds good. I wasn's sure about this one...